### PR TITLE
Fix History timestamps to follow --timestamp field.

### PR DESCRIPTION
**Pull request by rhatdan**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/buildah/issues/2589

**Comment by rhatdan** [comment]
@tensor5 PTAL
@nalind @TomSweeneyRedHat @vrothberg @giuseppe @ashley-cui @QiWang19 @mheon PTAL

File: imagebuildah/stage_executor.go [hunk 1]
```diff
@@ -789,8 +789,11 @@ func (s *StageExecutor) Execute(ctx context.Context, base string) (imgID string,
 				// for this stage.  Make a note of the
 				// instruction in the history that we'll write
 				// for the image when we eventually commit it.
-				now := time.Now()
-				s.builder.AddPrependedEmptyLayer(&now, s.getCreatedBy(node, addedContentSummary), "", "")
+				timestamp := time.Now()
```
Comment (nalind) [inline imagebuildah/stage_executor.go:792]: Force this to UTC, please.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
@nalind @vrothberg @giuseppe @ashley-cui @QiWang19 @mheon PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**